lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/ashrae_90_1_2019.AirLoopHVAC.rb

```diff
@@ -317,16 +317,60 @@ def air_loop_hvac_supply_air_temperature_reset_required?(air_loop_hvac, climate_
       return is_sat_reset_required
     end
 
+    # check if design outside air is less than 10,000cfm (5000L/s) 90.1 2019 6.5.3.5 Exception 1 and 2
+    design_oa_m3s = air_loop_hvac.sizingSystem.designOutdoorAirFlowRate.to_f
```

Comment:
The designOutdoorAirFlowRate method returns an optional double.  And this value is often autosized, so not set on the system before the system sizing is run.  The subsequent less than check will fail for most models.  Likewise, it is unclear why the follow if < 0.0001 statement is necessary - should the floor for this value be zero anyways?

Reply:
Refer to other HVAC controls methods that refer to the autosized values, such as `autosizedDesignSupplyAirFlowRate.is_initialized` in Standards.AirLoopHVAC.rb

Reply:
updated accordingly. 
(The original code was my improvised way of checking for autosized value, as if it is autosized, air_loop_hvac.sizingSystem.designOutdoorAirFlowRate.to_f gives 0, and I try not to put == between two floats, thus the 0.0001. Sorry for the confusions. Code is updated to same style as autosizedDesignSupplyAirFlowRate.is_initialized)

Reply:
> Also run rubocop on this file - seems like there may be a few uncaught errors.

I did not find rubocop errors in the code additions I made in this file. We could address rubocop problems in other parts in the big PR if you want.